src/plugins/googledrive/admin/src/index.js

```diff
@@ -0,0 +1,65 @@
+import { prefixPluginTranslations } from '@strapi/helper-plugin';
+import pluginPkg from '../../package.json';
+import pluginId from './pluginId';
+import Initializer from './components/Initializer';
+import PluginIcon from './components/PluginIcon';
+
+const name = pluginPkg.strapi.name;
+
+export default {
+  register(app) {
```

Comment:
This is for the admin dashboard. Can you add a comment saying we don't need to see the plugin in the dashboard, that's why this is commented?

Reply:
Is there any specific reason we don't need to see the google drive plugin in the dashboard? @jeyanthi-sm

Reply:
That was my suggestion to simplify the plugin. The plugin is essentially a proxy to Google Drive by authenticating with strapi instead of Google Drive. The data is stored in Google Drive. We don't really need to see the plugin in the admin dashboard.